garden_ai/app/notebook.py

```diff
@@ -270,29 +302,32 @@ def debug(
     Changes to the notebook file will NOT persist after the container shuts down.
     Quit the process with Ctrl-C or by shutting down jupyter from the browser.
     """
-    docker_client = docker.from_env()
-    base_image = _get_notebook_base_image(path) or "gardenai/base:python-3.10-jupyter"
 
-    if requirements_path is not None:
-        _validate_requirements_path(requirements_path)
+    with DockerClientSession() as docker_client:
```

Comment:
These aren't changes, just indents. Same for other blocks following `with DockerClientSession()`